salt/modules/solaris_shadow.py

```diff
@@ -190,6 +197,66 @@ def set_mindays(name, mindays):
     return False
 
 
+def gen_password(password, crypt_salt=None, algorithm='sha512'):
+    '''
+    .. versionadded:: 2014.7.0
```

Comment:
Will fix it once I get home today, I though I did it but must have forgotten it in the rebass to 2015.8.

Next release will be 2015.8.8?